packages/amplication-data-service-generator/src/server/message-broker/message-broker-service/create-message-broker-service.ts

```diff
@@ -0,0 +1,17 @@
+import { EventNames, Module } from "@amplication/code-gen-types";
+import pluginWrapper from "../../../plugin-wrapper";
+
+export async function createMessageBrokerServiceModules(): Promise<Module[]> {
+  return [
+    ...(await pluginWrapper(
+      async () => [],
+      EventNames.CreateMessageBrokerService,
+      {}
+    )),
+    ...(await pluginWrapper(
+      async () => [],
+      EventNames.CreateMessageBrokerServiceBase,
+      {}
+    )),
+  ];
```

Comment:
I think whether we should use a default structure for the plugin writer in this case - since we request the plugin to create a base file - I think we should provide a starting point. 
what do you think?